### DISCUSSION TITLE
How to get synthetic metric for a datetime range for a project ?

### DISCUSSION THREAD

**Discussion by gmaze** [green-coding-solutions/green-metrics-tool#915] [Q&A] [answered]
Hi,
How would it be possible with the API to retrieve the total energy used and CO2eq emissions for:
- a specific project uuid
- a specific datetime range

My goal is to be able to compute metrics associated with a specific software release

Thanks for your advice

**Comment by ArneTR** [comment, marked as answer]
Hey @gmaze 

we are currently in a bigger rewrite of CarbonDB.

As you might have noticed the current usability of CarbonDB is limited.

What we are introducing is a better filter system instead of the UUID. Similar to labels you have in AWS or Auzure for ressources.

This will also incorporate a time filter feature.

For the moment: We do not have an API endpoint that enables this.

What I can offer though is an export from the DB with all the data for your UUID in CSV format. Would that be helpful?

If so, ping me via email with your UUID. Otherwise open for other ideas if you have any.

The rewrite or CarbonDB will still take us 2-3 weeks I guess.

**Reply by gmaze** [reply]
thanks for your quick answer @ArneTR 
no hurry on my side, I can definitely wait for the new API !
